pkg/shell/cockpit-server.js

```diff
@@ -824,54 +849,120 @@ PageShutdownDialog.prototype = {
           $("#shutdown-action").click($.proxy(this, "restart"));
           $("#shutdown-action").text(_("Restart"));
         }
+
+        this.update_minutes();
         this.update();
     },
 
+    show_errors: function(show) {
+        $('#shutdown-parse-error').css('visibility', show ? 'visible' : 'hidden');
+        $('#shutdown-action').prop('disabled', show);
+        $("#shutdown-time").toggleClass("has-error", show);
```

Comment:
Please use the new dialog behavior functionality here: https://github.com/cockpit-project/cockpit/blob/master/pkg/playground/patterns.html#L93